docs/user/otp-deployment.md

```diff
@@ -0,0 +1,87 @@
+# Setting Up OTP Deployment Servers on Amazon Web Services (AWS)
+
+This document describes how to setup a load balancer server and an UI server for deployment using OTP DataTools, versions prior to Pull Request #506.
+
+Steps:
+
+1. Create the UI server (AWS S3 and CloudFront)
+2. Create the backend load balancer (AWS EC2).
+
+Assumptions for IBI-hosted deployments:
+
+1. You have access to an IBI Group Amazon Web Services (AWS) environment.
+2. Server instance types exist on AWS.
+3. A "key file name" exists.
+4. A Virtual Private Cloud (VPC) with two subnets exists on AWS.
+5. You are an admin for DataTools.
+
+## UI "Server": S3 Buckets and CloudFront
+
+### Create an S3 bucket
+
+Select AWS Console > Storage > S3 > Create bucket. Each deployment has its own bucket.
+When specifying options, uncheck 'Block All Public Access'.
+
+### Create a CloudFront instance
+
+1. Select AWS Console > Networking & Content Delivery > CloudFront > Create Ditribution. 
+2. Select Web Distribution, then click Next.
+3. Under Origin Settings, select the S3 bucket DNS name you created above.
+4. Under Cache Behavior, select 'Redirect HTTP to HTTPS'.
+5. Under Distribution Settings, select Custom SSL certificate, and select the ibi-transit.com certificate.
+6. Under Distribution Settings, set the Default Root Object value to be index.html.
+7. (Optional) Enter a comment to make the distribution easy to search. Leave other parameters as is, and click Create Distribution.
+8. Open the properties of this CloudFront instance, and copy the Domain Name value (e.g. **abcdef0123456789.cloudfront.net**).
+
+### Create a CNAME (i.e. subdomain) for CloudFront
+
+1. Select AWS Console > Networking & Content Delivery > Route 53 > Hosted Zones.
+2. Select ibi-transit.com.
+3. Select Create Record Set. Fill in the subdmain (e.g. **otp-mod-ui**.ibi-transit.com), and set the record type to **CNAME**.
+4. In the value field, paste the Domain Name value of the CloudFront instance above.
+5. Click Create.
+
+Once deployed, the OTP UI will be available at the URL configured in this section.
+
+## Backend server: Create a load balancer, and copy properties to DataTools
+
+We recommend using an elastic load balancer (ELB) for deploying/upgrading an OTP server instance without interrupting the current one. In doing so, a new server is set up in the background, and when ready (preprocessing done), the new server is added to the load balancer, and the old server removed and destroyed.
```

Comment:
Would be good to add language to this noting that multiple OTP servers can be instantiated/torn town.